src/memoize/memoizeProjectors.ts

```diff
@@ -1,5 +1,5 @@
-
-import { MapCache, memoize as lodashMemoize } from "lodash";
+import { MapCache } from "lodash";
```

Comment:
is there no equivalent in lodash-es or even an interface we can copy over? else maybe this should be like this instead, since we no longer depend on lodash
```suggestion
import type { MapCache } from "lodash";
```

Reply:
Unfortunately, it's only defined in `@types/lodash`: https://github.com/DefinitelyTyped/DefinitelyTyped/search?q=MapCache